src/cascadia/TerminalApp/Jumplist.cpp

```diff
@@ -118,8 +118,13 @@ static std::wstring_view _getExePath()
 // - settings - The settings object to update the jumplist with.
 // Return Value:
 // - <none>
-HRESULT Jumplist::UpdateJumplist(const CascadiaSettings& settings) noexcept
+winrt::fire_and_forget Jumplist::UpdateJumplist(const CascadiaSettings& settings) noexcept
 {
+    // make sure to copy the settings _before_ the co_await
+    const auto settingsCopy = settings.Copy();
+
+    co_await winrt::resume_background();
+

```

Comment:
whoa boy sorry, hold up

we don't need to _copy_ the settings. I apologize for my confusing wording.

We just need a _non-reference by-value capture_ of settings. A "Copy", in the C++ sense, of the value of `settings`. When you that with a C++/WinRT type, it increments the reference count and makes sure the reference count stays incremented until your local version is out of scope.

`const auto strongSettings = settings;`

Reply:
_Ah._ Thank you for the clarification, will remove the call to `Copy()`. 

I'm going to leave the ActionAndArgs copy fix in there though - I think that's a bug

Reply:
removed the copy fix to make a new PR for it